src/Benchmarks/Benchmarks.csproj

```diff
@@ -7,6 +7,7 @@
     <!-- TODO remove rid when https://github.com/dotnet/sdk/issues/396 is resolved -->
     <RuntimeIdentifier Condition="'$(TargetFramework)'!='netcoreapp1.1'">win7-x64</RuntimeIdentifier>
```

Comment:
Instead, consider deleting this line and the TODO before. This is the bug that was resolved.